lldb/unittests/Host/posix/HostTest.cpp

```diff
@@ -115,5 +115,8 @@ TEST_F(HostTest, GetProcessInfoSetsPriority) {
   }
   ASSERT_TRUE(Info.IsZombie().has_value());
   ASSERT_FALSE(Info.IsZombie().value());
+
+  ASSERT_TRUE(Info.IsCoreDumping().has_value());
+  ASSERT_FALSE(Info.IsCoreDumping().value());
```

Comment:
I presume this field has been around for a very long time, we don't expect any system to not have it?

Reply:
Since 4.15, so around 2018 was the end of LTO for that version. I think we're safe to assume it's going to be ubiquitous on our test hosts. As for customers, we would just fail to read it